python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -473,15 +486,21 @@ def __call__(self, measure_input, tmp_dir, **kwargs):
         return BuildResult(filename, arg_info, None, time.time() - tic)
 
 
+CodeLoader = typing.Callable[
+    [dict, dict], typing.ContextManager[typing.Tuple[tvm.rpc.RPCSession, tvm.runtime.Module]]
+]
+
+
 def run_through_rpc(
     measure_input,
     build_result,
     number,
     repeat,
     min_repeat_ms,
     cooldown_interval,
-    remote_args,
+    remote_kw,
```

Comment:
Better to use full name `remote_kwargs`?

Reply:
also a good point, done